build/common/installer/scripts/tomlparser-agent-config.rb

```diff
@@ -186,6 +191,32 @@ def populateSettingValuesFromConfigMap(parsedConfig)
           puts "Using config map value: NODES_EMIT_STREAM_BATCH_SIZE = #{@nodesEmitStreamBatchSize}"
         end
       end
+
+      # networkflow logs settings
+      networkflow_config = parsedConfig[:agent_settings][:networkflow_config]
+      if !networkflow_config.nil?
+        netoworkFlowLogsThrottleRate = networkflow_config[:throttle_rate]
+        if is_valid_number?(netoworkFlowLogsThrottleRate)
```

Comment:
create a task to do the lower and upper bounds. if its either lower than lower bound or higher than upper bound, use the defaults.